fieldservice_sale_stock/models/sale_order.py

```diff
@@ -30,8 +30,11 @@ def _link_pickings_to_fsm(self):
                 rec.procurement_group_id.fsm_order_id = fsm_order.id or False
             for picking in rec.picking_ids:
                 picking.write(rec.prepare_fsm_values_for_stock_picking(fsm_order))
-                for move in picking.move_lines:
-                    move.write(rec.prepare_fsm_values_for_stock_move(fsm_order))
+                for move in picking.move_ids:
+                    for move_line in move._get_move_lines():
+                        move_line.write(
+                            rec.prepare_fsm_values_for_stock_move(fsm_order)
+                        )
```

Comment:
is there any reason to write on `stock.move.line` when previously it was written on `stock.move`?
v15 `move_lines` field on `stock.picking` was just renamed to `move_ids` in v16